### PR TITLE
Grammar & Clarification

### PR DESCRIPTION
**Pull request by ericelliott**
Just taking a pass at grammar issues, and trying to clarify some points.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -67,22 +53,20 @@ The stamp descriptor properties are made available on each stamp as `stamp.compo
...
 Descriptors are composed together to create new descriptors with the following rules:
 
-* `methods` are shallow mixed: `descr.methods = _.assign({}, descr1.methods, descr2.methods)`
-* `properties` are shallow mixed: `descr.properties = _.assign({}, descr1.properties, descr2.properties)`
-* `deepProperties` are deep merged: `descr.deepProperties = _.merge({}, descr1.deepProperties, descr2.deepProperties)`
-* `initializers` are stacked: `descr.initializers = descr1.initializers.concat(descr1.initializers)`
-* `staticProperties` are shallow mixed: `descr.staticProperties = _.assign({}, descr1.staticProperties, descr2.staticProperties)`
-* `propertyDescriptors` are deep merged: `descr.propertyDescriptors = _.merge({}, descr1.propertyDescriptors, descr2.propertyDescriptors)`
-* `configuration` are deep merged: `descr.configuration = _.merge({}, descr1.configuration, descr2.configuration)`
+* `methods` are copied by assignment: `stamp.methods = _.assign({}, stamp1.methods, stamp2.methods)`
```
Comment (koresar) [inline README.md]: `stamp` does not have `.methods`, `.properties`, etc. Only descriptor has those.
Stamp can have only `.compose`.

Confused yourself to stampit? :)

File: README.md [hunk 2]
```diff
@@ -67,22 +53,20 @@ The stamp descriptor properties are made available on each stamp as `stamp.compo
 * `methods` - A set of methods that will be added to the object's delegate prototype.
 * `properties` - A set of properties that will be added to new object instances by assignment.
 * `deepProperties` - A set of properties that will be added to new object instances by assignment with deep property merge.
-* `initializers` - A set of functions that will run in sequence and passed the data necessary to initialize a stamp instance.
+* `initializers` - A set of functions that will run in sequence. Stamp details and arguments get passed to initializers.
```
Comment (koresar) [inline README.md]: Great addition!

File: README.md [hunk 3]
```diff
@@ -40,23 +36,13 @@ It's a collection of properties that contain the information necessary to create
 * `compose(...stampsOrDescriptors) => stamp` **Creates stamps.** Take any number of stamps or descriptors.
 Return a new stamp that encapsulates combined behavior. If nothing is passed in, it returns an empty stamp.
 
------
```
Comment (koresar) [inline README.md]: I would prefer to educate people based on examples. Can you please move this "Similarities" to, probably, end this file or another file or something?

**Comment by koresar** [comment]
Other than error in "Desription" section I love the changes.

File: README.md (hunk 1 above)
Comment (ericelliott) [inline README.md]: Good point. =)

File: README.md (hunk 3 above)
Comment (ericelliott) [inline README.md]: Ah, I knew there was something I meant to move, but never found a place for... Good catch!

**Comment by ericelliott** [comment]
Great. Incorporating your suggestions... =)

**Comment by koresar** [comment]
LGTM

**Comment by JosephClay** [comment]
Good updates @ericelliott. It's a much easier read.

**Pull request merged**